doc/README.md

```diff
@@ -0,0 +1,11 @@
+Quick start guide:
+
+- Install Hugo; follow the guidance on https://gohugo.io/getting-started/installing.
+  You'll need Go as well: see https://go.dev/
```

Comment:
Go itself is only needed because of the way I have set up the theme using "modules"; see https://gohugo.io/hugo-modules/use-modules. The alternative is to copy the theme repo in a `/themes` directory, or use git submodules, but I think those options are less practical, and not the preferred way of doing it anymore.